servo-git/PKGBUILD

```diff
@@ -56,7 +57,8 @@ prepare() {
 
 build() {
 	cd "$pkgname"
-	export RUSTUP_TOOLCHAIN=stable
+	# 2024-12-20: crown FTBFS with rust 1.83.0
```

Comment:
Is there an upstream issue to track this that we could link here so it would be easy to confirm whether this workaround is ready for removal or not?

Reply:
I couldn't find one.
Do you think it makes more sense to just remove these lines so it automatically picks right version from `rust-toolchain.toml`?

Reply:
If there is a `rust-toolchain.toml` file, then yes we should let it pick that up, with the caveat that it will only pick that up if we're building with Rust tooling supplied by `rustup` ... which in this case we are.

Reply:
Done